photutils/morphology.py

```diff
@@ -5,14 +5,27 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
+import collections
 import numpy as np
-from astropy.modeling import models
+from astropy.modeling.models import Gaussian1D, Gaussian2D, Const1D, Const2D
 from astropy.modeling.fitting import LevMarLSQFitter
+from astropy.nddata.utils import overlap_slices
 from .segmentation import SegmentProperties
+import warnings
 
 
-__all__ = ['centroid_com', 'gaussian1d_moments', 'centroid_1dg',
-           'centroid_2dg', 'fit_2dgaussian', 'data_properties']
+__all__ = ['GaussianConst1D', 'GaussianConst2D', 'centroid_com',
```

Comment:
Should we expose `GaussianConst1D` and `GaussianConst2D` in the docs (i.e. kind of make them part of our public API)?
The main issue with this I see is that the docstring doesn't even mention the parameter names (or does it via some Python trickery?), i.e. makes this model hard to use.